CRD-DTR/TIRF/R4/resources/Questionnaire-R4-TIRF.json

```diff
@@ -161,46 +156,42 @@
             "required": true,
             "answerOption": [
               {
-                "valueString": "Text to Mobile #"
+                "valueCoding": {
+                  "code": "Text to Mobile #",
```

Comment:
the codes for these can be found here: https://build.fhir.org/valueset-contact-point-system.html